config/schema/sharethis.schema.yml

```diff
@@ -82,3 +81,12 @@ sharethis.settings:
         hashaddress:
           type: integer
           label: 'Has Address'
+    sharethisnodes:
+      type: sequence
+      label: 'Node Option Modes'
+      sequence:
+        type: sequence
```

Comment:
Instead of sequence can we do config.entity_view_display. [%parentid]
It will provide us free validation to us and will keep only existing content types